### ISSUE TITLE
[proposal] Add basic support for ARIA Accessibility Options

### ISSUE THREAD

**Issue by mmikalsen** [go-echarts/go-echarts#530]
### Describe your proposal

Currently, go-echarts lacks built-in support for generating ARIA (Accessible Rich Internet Applications) descriptions, making charts less accessible to users who rely on screen readers and other assistive technologies. Apache ECharts already implements this [feature](https://echarts.apache.org/en/option.html#aria), automatically generating chart descriptions based on on the chart, data, title, etc..

### What the feature/improvement it should looks like?

Implement ARIA support mirroring Apache ECharts' implementation with:
```go
// Aria is the option set for the Aria component.
// https://echarts.apache.org/en/option.html#aria
type Aria struct {
	// Whether to enable the ARIA accessibility (web accessibility) of the chart.
	// When enabled, the description of the chart will be automatically and
	// intelligently generated based on the chart, data, title, etc.
	Enabled types.Bool `json:"enabled,omitempty"`
	Label   AriaLabel  `json:"label,omitempty"`
}

// AriaLabel is the options set for aria Label.
// https://echarts.apache.org/en/option.html#aria.label
type AriaLabel struct {
	// Overwrite the automatic generated aria-label.
	Description string `json:"description,omitempty"`
}

```

usage example:
```go
	bar := charts.NewBar()
	bar.SetGlobalOptions(charts.WithTitleOpts(opts.Title{
		Title:    "bar chart with aria",
		Subtitle: "aria supported bar chart",
	}),
		charts.WithAriaOpts(opts.Aria{Enabled: opts.Bool(true)}),
	)
```

### Willing to implement it?

- [ ] Yes, I do

**Comment by Koooooo-7** [comment]
Hi @mmikalsen , I add the basic support for the `aria`.
But I encounter some issue for those options (it seems not effect well).
You could have a try and wait for your feedback.
Thx!

**Comment by mmikalsen** [comment]
Thank you for implementing it 🎉
It seems good enough for me, enabling the auto generated aria-label and setting a custom one works as expected
